app/views/coffees/_roaster_coffee_card.html.erb

```diff
@@ -1,82 +1,26 @@
-    <%= render 'shelf_items/shelf_item_toggle', coffee: coffee %>
-  </div>
-
-  <div class="tasting-notes">
-    <h2>
-      <strong>Tasting notes:</strong>
-      <%= coffee.tasting_notes %>
-    </h2>
-  </div>
-
-  <div class="website">
-    <a href="<%= coffee.url %>" target=_blank>
-      View on <%= coffee.roaster.name %>'s website<i class="las la-external-link-alt"></i>
-    </a>
-  </div>
-
-  <div class="stats">
-    <div class="brews">
-      <%= coffee.brews_count %> brews logged
-    </div>
-    <div class="shelf_items">
-      <%= coffee.shelf_items_count %> people currently have this coffee
-    </div>
-    <div class="favourites">
-      <%= coffee.favourites_count %> people have this saved as one of their favourites
-    </div>
-    <div class="reviews">
-      There are <%= coffee.reviews_count %> reviews of this coffee.
-    </div>
-  </div>
-
-  <div class="details-container">
-    <div class="details">
-      <h2>Details</h2>
+<div id="<%= dom_id coffee %>" class="coffee-card card column">
+  <div class="coffee-card-content row">
+    <div class="column">
+      <h3 class="name"><%= link_to coffee.name, coffee, class: 'link link-primary link-primary-title' %></h3>
       <div class="country">
-        <strong>Country:</strong>
-        <%= coffee.country %>
-      </div>
-      <div class="region">
-        <strong>Region:</strong>
-        <%= coffee.region %>
-      </div>
-      <div class="town">
-        <strong>Town:</strong>
-        <%= coffee.town %>
-      </div>
-      <div class="altitude">
-        <strong>Altitude:</strong>
-        <%= coffee.altitude %> masl
+        <i class="las la-map-marker"></i> <%= (coffee.country.present? ? coffee.country : coffee.region.present? ? coffee.region : 'Unknown').titleize %>
       </div>
-      <div class="process">
-        <strong>Process:</strong>
-        <%= coffee.process %>
-      </div>
-      <div class="variety">
-        <strong>Variety:</strong>
-        <%= coffee.variety %>
-      </div>
-      <div class="producer">
-        <strong>Producer:</strong>
-        <%= coffee.producer %>
+      <div class="tasting-notes">
+        <i class="las la-music"></i> <%= coffee.tasting_notes.titleize %>
       </div>
     </div>
+    <div class="controls column">
+      <%= render 'favourites/favourite_toggle', coffee: coffee %>
+      <%= render 'shelf_items/shelf_item_toggle', coffee: coffee %>
+    </div>
   </div>
-
-  <div class="coffee-link">
-    <%= link_to "Show this coffee", coffee %>
+  <div class="metadata row">
+    <span><i class="las la-mug-hot"></i> <%= coffee.brews_count %></span>
+    <span>&#8226;</span>
+    <span><i class="las la-hand-holding"></i> <%= coffee.shelf_items_count %></span>
+    <span>&#8226;</span>
+    <span><i class="las la-heart"></i> <%= coffee.favourites_count %></span>
+    <span>&#8226;</span>
+    <span><i class="las la-comment"></i> <%= coffee.reviews_count %></span>
```

Comment:
I think these need tooltips

Reply:
For sure. Logos will need reviewing too. They’re just placeholders for now. Considering adding the separating dots using CSS instead of in HTML.

Reply:
one for now or one for a future ticket?

Reply:
also, do we want to make these live updating? not sure how much performance impact that will have so maybe one for later

Reply:
I assumed they would live update until I tested it. Maybe we don't need it live updating globally, but if a user clicks the favorite button, they should see their client update the count by 1.

Reply:
I think we could probably get them globally updating more easily than local